tests/test_daemon.cpp

```diff
@@ -780,4 +879,121 @@ INSTANTIATE_TEST_SUITE_P(Daemon, LaunchImgSizeSuite,
                                  Values(std::vector<std::string>{}, std::vector<std::string>{"--disk", "4G"}),
                                  Values("1G", mp::default_disk_size, "10G")));
 
+class NotEmptyStubVMImageVault final : public mpt::StubVMImageVault
+{
+    bool has_record_for(const std::string& name) override
+    {
+        return name == "real-zebraphant";
+    }
+};
```

Comment:
Did you consider just using the `MockVMImageVault` instead?

Reply:
No, I didn't and I realized that it was much simpler. Changed.